content/docs/studio/user-guide/prepare-your-repositories.md

```diff
@@ -6,26 +6,15 @@
 
 # Prepare Your Repositories
 
-Iterative Studio creates [projects] by identifying machine learning models,
-datasets, metrics and hyperparameters defined in your Git repositories. These
-values are found in JSON or YAML files in the repository. Additionally, model
-information may be available as Git tags.
-
-New ML model metadata can be added directly from the **Models** tab after
-[creating a project]. You can also use [GTO] or [MLEM].
-
-[projects]: /doc/studio/user-guide/projects-and-experiments/what-is-a-project
-[creating a project]:
-  /doc/studio/user-guide/projects-and-experiments/create-a-project
-[gto]: https://github.com/iterative/gto
-[mlem]: https://mlem.ai/
-
-Datasets, metrics, and hyperparameters can be added to a project in two ways:
-
-1. **Set up DVC repositories**: You can use [DVC](https://dvc.org/) and Git to
-   version your code, data and models all within your Git repositories. Data
-   Version Control, or DVC, is a data and ML experiment management tool that
-   takes advantage of the existing engineering toolset that you're already
+Iterative Studio creates [projects][project] by identifying machine learning
+models, datasets, metrics and hyperparameters defined in your Git repositories.
+These values can be added to a project in two ways:
+
+1. **Set up DVC in your repositories**: You can use [DVC](https://dvc.org/) and
+   Git to version your code, data and models all within your Git repositories.
+
+   Data Version Control, or DVC, is a data and ML experiment management tool
+   that takes advantage of the existing engineering toolset that you're already
    familiar with (Git, CI/CD, etc.). By using DVC, you can be sure not to bloat
```

Comment:
Small clarification (less repetitive IMO); And avoid documenting DVC here. Minor (and PR-unrelated)

```suggestion
1. **Set up DVC in your repositories**: You can use [DVC](https://dvc.org/) and
   Git to version your code, data, and models from your project repositories. 
   This avoids bloating
```